plans/all.fmf

```diff
@@ -0,0 +1,46 @@
+adjust:
+    when: initiator is not defined or initiator != packit
+    discover+:
+        dist-git-source: true
+
+discover+:
```

Comment:
nitpick, I usually don't put `+` on the lowest level, just to make sure I am not inheriting from something I don't expect, e.g. forgetting a `main.fmf` in the source.